app/models/project.rb

```diff
@@ -0,0 +1,3 @@
+class Project < ApplicationRecord
+  validates :urn, presence: true
```

Comment:
Given our level of confidence about this being an integer, what do you think about validating that as well?